Intermediate-concurrency-tutorial/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Multithreading introductory task in Dell mentoring program.
+
+### How to execute program:
+1. Please, create executable jar file of this program. Instruction might be found in this link: https://www.jetbrains.com/help/idea/packaging-a-module-into-a-jar-file.html
```

Comment:
1. It'd be easier for the users to copy-paste the command and get a jar right away rather than going through an instruction on an external website.
2. There could be issues with the website link: it could become unavailable and/or outdated. How'd user have to figure out the process of building the jar file? 
3. Here, you're forcing users to build jar though and IDE. Some users might have reasons to not use the Idea, others wouldn't have the ability to do so since they could use Linux without any GUI.  

Please, consider adding a command or Gradle task to build a jar file.